### PR TITLE
fix: custom colors in dark mode for logo's

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
In a previously PR we fixed a few things related to dark mode and logo's, but also introduced new bugs. Additionally, we add a few small improvements to some of the logo's in the footer for a better presentation.

### CODE REVIEW

File: libs/base/ui/graphical/image/src/image.styles.ts [hunk 1]
```diff
@@ -32,9 +32,9 @@ export const styles = css`
     object-position: var(--image-position);
   }
 
-  @media (prefers-color-scheme: dark) {
+  /* @media (prefers-color-scheme: dark) {
```
Comment (tolerants) [inline libs/base/ui/graphical/image/src/image.styles.ts]: Can we just remove it?

File: libs/platform/experience/src/services/layout/plugins/styles/canvas/canvas-style.model.ts [hunk 2]
```diff
@@ -6,6 +6,13 @@ declare global {
      */
     background?: string;
 
+    /**
+     * Sets the overflow behavior of an element's content. If an element's content is larger than
```
Comment (tolerants) [inline libs/platform/experience/src/services/layout/plugins/styles/canvas/canvas-style.model.ts:10]: Does it really make sense to explain css properties? In any case we are not providing possible values and not affecting default implementation in the browser. Maybe we can just give a link to MDN or W3?

**Review by tolerants — CHANGES_REQUESTED** [review]

File: libs/platform/experience/src/services/layout/plugins/styles/canvas/canvas-style.model.ts (hunk 2 above)
Comment (tobi-or-not-tobi) [reply libs/platform/experience/src/services/layout/plugins/styles/canvas/canvas-style.model.ts:10]: We had it on other elements and hence I stayed consistent with it. The longer term vision is that we'd have docs on those attributes for both devs and users in EB. Inline docs is not working for that, I know.

Comment (tobi-or-not-tobi) [reply libs/platform/experience/src/services/layout/plugins/styles/canvas/canvas-style.model.ts:10]: lets keep it for now

**Review by tolerants — APPROVED** [review]

**Pull request merged**